test/CMakeLists.txt

```diff
@@ -1,7 +1,17 @@
 find_package(ament_cmake_gtest REQUIRED)
 find_package(ament_cmake_pytest REQUIRED)
 
-ament_add_pytest_test(maliput_to_obj_test maliput_to_obj_test.py WORKING_DIRECTORY ${CMAKE_INSTALL_PREFIX}/bin TIMEOUT 360)
+
+# TODO(https://github.com/maliput/maliput_infrastructure/issues/222): Use ament_index for finding resources provided by the packages.
+set(MALIPUT_MULTILANE_RESOURCE_PATH "${maliput_multilane_DIR}/..")
+set(MALIPUT_MALIDRIVE_RESOURCE_PATH "${maliput_malidrive_DIR}/..")
+
+ament_add_pytest_test(maliput_to_obj_test maliput_to_obj_test.py
+  WORKING_DIRECTORY ${CMAKE_INSTALL_PREFIX}/bin TIMEOUT 360
+  ENV "MULTILANE_RESOURCE_ROOT=${MALIPUT_MULTILANE_RESOURCE_PATH}"
+)
+
+message(STATUS "****** maliput_multilane: ${maliput_multilane_DIR}")
```

Comment:
Is this message intended?

Reply:
Ups, it's a leftover of some testing.
done.